lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -47,7 +33,8 @@
     ad_user=dict(type='str', no_log=True),
     password=dict(type='str', no_log=True),
     cloud_environment=dict(type='str'),
-    cert_validation_mode=dict(type='str', choices=['validate', 'ignore'])
+    cert_validation_mode=dict(type='str', choices=['validate', 'ignore']),
+    api_profile=dict(type='str', default='latest')
     # debug=dict(type='bool', default=False),
```

Comment:
why should api_profile be module parameter?

Reply:
The intent is to allow it to be set/overridden the same way as all other "auth-y/connection-y" stuff, so we'd also add it to env and disk cred profiles once it's fully-supported (also important for Tower). I'm not sure if the CLI stuff exposes the selected api_profile the same way as current cloud_env/current subscription/etc, but we'd need a way to get that when using CLI auth (eg against an Azure Stack instance).